.github/workflows/release.yaml

```diff
@@ -43,15 +45,8 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
-      - name: Create release pull request or publish to npm
-        id: changesets
-        uses: changesets/action@v1
-        with:
-          version: pnpm ci:version
-          publish: pnpm ci:release
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      - run: node scripts/prepublish.mjs
+      - run: pnpm publish --access public
```

Comment:
package.json already contains this flag.
```json
"publishConfig": {
        "access": "public"
    }
```

Reply:
Also Github release will be missing, and so will Changelog.md changes.

Reply:
In a personal conversation with @generall we decided to skip github releases. It contains changelog and zip sources only, I don't see good reasons to continue to support it